src/lib.rs

```diff
@@ -139,14 +141,15 @@ impl Progress {
         let label: String = label.into();
 
         // An initial "empty" rendering of the new bar.
-        println!(
+        writeln!(
+            &mut self.out,
             "{:<l$}      [{:->f$}]   0%",
             label,
             "",
             l = twidth - w - 8 - 5,
             f = w
-        );
-        self.out.flush().unwrap();
```

Comment:
The `flush` was removed here, but kept elsewhere. What was the intent?

Reply:
Since we call `writeln!` and `self.out` is line-buffered it'll flush anyway. The other cases don't write a newline so an explicit flush is needed.